tools/bin/ecflow/processes/main/meteorological/get_metdata.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+#####get_metdata.py 
+#####usage: <python> <get_metdata.py> <configuration.cfg>
+
+###This script downloads meteorological data through xarray from
+###http://thredds.northwestknowledge.net:8080/thredds/reacch_climate_MET_catalog.html
+###delivered through OPeNDAP.
+
```

Comment:
Basically, this script would benefit from being split into a few pieces, wrapped in a `main` function. As an example, I'd write a series of separate functions:

- `main()` that calls: 
  - `parse_command_line()`
  - `make_metadata_attrs()`
  - `read_data()`
  - `set_metadata_attrs()`
  - `write_data()`